tracer/test/test-applications/debugger/Samples.Probes/SmokeTests/AsyncThrowExceptionAsyncVoid.cs

```diff
@@ -1,22 +0,0 @@
-using System;
-using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
-
-namespace Samples.Probes.SmokeTests
-{
-    internal class AsyncThrowExceptionAsyncVoid : IAsyncRun
-    {
-        public async Task RunAsync()
-        {
-            await Task.Run(() => { VoidMethod(nameof(RunAsync)); });
-        }
-
-        [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
-        [MethodProbeTestData(skip:true)]
-        private async void VoidMethod(string caller)
-        {
-            await Task.Delay(20);
-            throw new InvalidOperationException($"Exception from {caller}.{nameof(VoidMethod)}");
-        }
-    }
```

Comment:
Why you've deleted this and not simply avoid throwing the exception?

Reply:
Chatted on Zoom, Dudi explained to me that he has another test that is identical then he decided to get rid of it.
Resolving.